converter/src/main/java/gov/cms/qpp/conversion/encode/JsonWrapper.java

```diff
@@ -401,7 +401,12 @@ protected int validInteger(String value) {
 	 */
 	protected String validDate(String value) {
 		try {
-			LocalDate thisDate = LocalDate.parse(cleanString(value),  DateTimeFormatter.ofPattern("yyyyMMdd"));
+			String parse = cleanString(value);
+			parse = parse.replace("-", "").replace("/", "");
+			if (parse.length() > "yyyyMMdd".length()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "yyyyMMdd" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)

Reply:
I agree with @FlexionSonarQube here.

Reply:
Working on it